go.mod

```diff
@@ -3,24 +3,45 @@ module github.com/cloudability/metrics-agent
 go 1.14
 
 require (
+	github.com/Azure/azure-sdk-for-go v43.0.0+incompatible // indirect
+	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20200415212048-7901bc822317 // indirect
+	github.com/Microsoft/hcsshim v0.8.10-0.20200715222032-5eafd1556990 // indirect
+	github.com/containernetworking/cni v0.8.0 // indirect
+	github.com/coredns/corefile-migration v1.0.10 // indirect
+	github.com/docker/docker v1.4.2-0.20200309214505-aa6a9891b09c // indirect
+	github.com/evanphx/json-patch v4.9.0+incompatible // indirect
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golangci/golangci-lint v1.23.8
-	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/google/cadvisor v0.37.0 // indirect
+	github.com/googleapis/gnostic v0.4.0 // indirect
 	github.com/gostaticanalysis/analysisutil v0.0.3 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
+	github.com/ishidawataru/sctp v0.0.0-20190723014705-7c296d48a2b5 // indirect
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/moby/ipvs v1.0.1 // indirect
 	github.com/onsi/gomega v1.8.1
-	github.com/sirupsen/logrus v1.4.3-0.20190701143506-07a84ee7412e
-	github.com/spf13/cobra v0.0.5
+	github.com/opencontainers/runc v1.0.0-rc91.0.20200707015106-819fcc687efb // indirect
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spf13/cobra v1.0.0
```

Comment:
Hi @bbylovas hope you are doing well! is this compatible with https://github.com/cloudability/metrics-agent/blob/3789703ff29a49967d5e378098e3c6f5fea19187/util/util.go#L73-L90 ?

Reply:
Heh nvm it is 👍